e2e-tests/dotnet-rest/test/RestApis.Tests.HelloWorld/Controller/EmbeddedEvoMasterController.cs

```diff
@@ -51,7 +50,7 @@ public class EmbeddedEvoMasterController : EmbeddedSutController {
                 RestApis.HelloWorld.Program.Main (new string[] { ephemeralPort.ToString () });
             });
 
-            await WaitUntilSutIsRunningAsync (ephemeralPort);
+            WaitUntilSutIsRunning (ephemeralPort);
```

Comment:
we do not want to wait forever, in case of problems. so, should be an option to state of how long we wait at most (eg 20 seconds) before giving up

Reply:
I added a timeout argument to this method with a default value of 20 seconds. If the SUT didn't start within the specified amount of time, a TimeoutException will be thrown